packages/components/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@auth0/styleguide-components",
-  "version": "2.0.10",
+  "version": "2.0.11",
```

Comment:
I'm assuming that merging this will publish https://cdn2.auth0.com/styleguide/components/2.0.11/components.min.css

Reply:
May not make a difference, but looks like other PRs handle package bumps as a separate commit (see https://github.com/auth0/styleguide/pull/217). May want to redo the PR just in case?

Reply:
Good point, done